src/ape/managers/accounts.py

```diff
@@ -46,8 +46,8 @@ def containers(self) -> dict[str, TestAccountContainerAPI]:
             t for t in self.plugin_manager.account_types if issubclass(t[1][1], TestAccountAPI)
         ]
         for plugin_name, (container_type, account_type) in account_types:
-            # Pydantic validation won't allow passing None for data_folder/required attr
-            containers[plugin_name] = container_type(data_folder="", account_type=account_type)
+            # Pydantic validation won't allow passing None for name/required attr
+            containers[plugin_name] = container_type(name="", account_type=account_type)
```

Comment:
Can this be like `name=f"<{plugin_name}>"` or something just to make sure if never overwrites anything?

Reply:
This made me realize this needs change. Here is my plan, let me know what you think:

1. Make this `name=plugin_name` and always provide a container name. Make it _actually_ required.
2. Allow containers to change the data folder again but don't make it be required; have it default to `.ape/self.name`. But allowing it still to change is helpful in the case a plugin has multiple API classes; it could nest the accounts under `.ape/plugin_name/accounts` or something alike.

Doing it now and will link the commit, might be easier to see what I mean.

Reply:
OK tad different but: https://github.com/ApeWorX/ape/pull/2025/commits/0965c50a3a9a04a5f2568ba012e5214463b9d318

1. Uses plugin_name for name always, event test containers
2. for test containers, point data_folder at /dev/null... It is not used anywhere and this achieves the same idea and avoid typing AND pydantic concerns because it is still a path at least.

Reply:
and if any plugin can change this via overriding the property

Reply:
I think this works great - still allows overriding, better null-value for the type, no actual changes to any plugin besides the field change.

Let me know if have any other ideas and ill edit-rebase them in!